storage.bs

```diff
@@ -179,6 +179,10 @@ defined by the following table:
   <td>"<code>caches</code>"
   <td>« "<code>local</code>" »
   <td>null
+ <tr>
+  <td>"<code>fileSystem</code>"
+  <td>« "<code>local</code>" »
+  <td>5 &times; 2<sup>20</sup> (i.e., 5 mebibytes)
```

Comment:
This is currently defined as `null` in the FS spec, but I think that should be updated? https://fs.spec.whatwg.org/#sandboxed-filesystem